src/plugins/intel_cpu/tests/functional/single_layer_tests/interpolate.cpp

```diff
@@ -187,17 +188,15 @@ class InterpolateLayerCPUTest : public testing::WithParamInterface<InterpolateLa
         std::vector<int64_t> axes;
         std::tie(shapeCalcMode, dataShape, shapeInputType, shapeDataForInput, axes) = shapeParams;
 
-        if (shapeCalcMode == ngraph::op::v4::Interpolate::ShapeCalcMode::SCALES) {
+        if (shapeCalcMode == ngraph::op::v11::Interpolate::ShapeCalcMode::SCALES) {
```

Comment:
Please use ov namespace where it's possible.

Reply:
done, thanks